README.md

```diff
@@ -50,6 +50,22 @@ The official Sift .NET client, supporting .NET Standard 2.0+
         // Handle InnerException
     }
 
+#### IncludeScorePercentile
+     public bool IncludeScorePercentile { get; set; }
+
+#### Url Construction
+     protected override Uri Url
+     {
+            get
+            {
+                var url = new Uri(EventsUrl);
+                if (IncludeScorePercentile)
+                {
+                    url = url.AddQuery("fields", "SCORE_PERCENTILES");
+                }
+            }
+     }
+
```

Comment:
No need to reflect this in README, instead - let's just provide a comment (or example) how we should modify the `EventRequest` in order to support score percentiles